### PR TITLE
Added import possibilities

### PR DESCRIPTION
**Pull request by jbusecke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -252,9 +253,46 @@ get started, assuming that your are working in the Resplandy group,
...
+
+3. ... other methods are not documented yet -->
+
+<a id="down_ssh"></a>
+## Download to local machine and transfer to remote
+Download your dataset to a location on your harddrive (e.g. `~/Downloads`).
+
+From there you can copy the file to the remote filesystem by using
+```
+scp ~/Downloads/<yourfile> <username>@tigressdata.princeton.edu:/tigress/<username>/`
```
Comment (luet) [inline README.md]: There is an extraneous ` at the end of that line

File: README.md [hunk 2]
```diff
@@ -252,9 +253,46 @@ get started, assuming that your are working in the Resplandy group,
...
+```
+The words in `<...>` need to be replaced with specific filesnames and your princeton username.
+If you have set up SSH keys (e.g. if you log into `tigressdata` with `ssh tigressdata`),
+you can simplify the command above to:
+```
+scp ~/Downloads/<yourfile> tigressdata:/tigress/<username>/`
```
Comment (luet) [inline README.md]: same problem here

**Review by luet — CHANGES_REQUESTED** [review]

**Comment by luet** [comment]
How is the user downloading the data on their own machine? Is through a web browser? If so why not use a web browser on tigressdata through TurboVNC?

**Comment by jbusecke** [comment]
Yeah, that is a valid point. But that will be very different in each case. I actually don't know how to do that via TurboVCN. I might learn something hahaha

**Comment by jbusecke** [comment]
Is this still of interest? Or should I close?

**Comment by cses** [comment]
I added the section in this PR in commit 12707bf.

**Comment by luet** [comment]
I added the section in this PR in commit 12707bf.

**Pull request closed without merging**